cloudformation/media-atoms-dynamo-code.json

```diff
@@ -15,8 +15,12 @@
             "Description": "Code audit dynamo table",
             "Type": "String",
             "Default": "audit-media-atom-maker-CODE"
+        },
+        "UploadTrackingAtomMakerCodeTable": {
```

Comment:
do we want to bite the bullet now and put new dynamo tables in the main cfn template? cc @jennysivapalan